hm-rest-auto-api-v4.yml

```diff
@@ -2,7 +2,7 @@ info:
     title: 'HM REST Auto API'
     version: 4.0.0
     description: 'High Mobility REST Auto API Level 12 specification'
-swagger: '2.0'
+swagger: '3.0'
```

Comment:
@kaareloun getting this error when previewing the swagger in https://editor.swagger.io/

"Please indicate a valid Swagger or OpenAPI version field. Supported version fields are swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0)."

This should fix it. But is it now in the new format?

``openapi: '3.0.0'``